README.md

```diff
@@ -1,14 +1,54 @@
+
 # DexScript - ALPHA
+<img src="https://i.imgur.com/uKfx0qO.png" width="200"> 
 
-DexScript is a set of commands developed by DotZZ on September 14, 2024, used for **BallsDex and CarFigures** Discord bot forks.
-Below is a guide on how to use DexScript, and what DexScript is capable of.
-Currently, DexScript is in alpha. DexScript will have more commands, the ability to support more models, and overall quality-of-life improvements as time passes.
+## What is DexScript?
+DexScript is a set of commands made by ``DotZZ``. The commands simplify things like editing, adding, and deleting things you want in your bot. Basically making your life easier.
```

Comment:
I would recommend changing "things" to models

Reply:
Will do.